lib/improver/cli/__init__.py

```diff
@@ -160,6 +160,19 @@ def inputjson(to_convert):
     return maybe_coerce_with(load_json_or_none, to_convert)
 
 
+@value_converter
+def comma_separated_list(to_convert):
```

Comment:
`comma_separated_list` -> `csvlist`?

Reply:
I thought about that, but I was concerned that people would get it confused with the file type, rather than the concept. I'm happy to change it though.

Reply:
I'm happy with this, it's clear